tests/monitoring_integration_test.py

```diff
@@ -106,6 +106,56 @@ def testUserExpDashboard(self):
 				
 		self.assertTrue(found_userexp_dashboard)
 
+class TestCustomService(unittest.TestCase):
+	def setUp(self):
+		self.client = monitoring_v3.ServiceMonitoringServiceClient()
+		self.project_id = getProjectId()
+
+	def checkForService(self, service_name):
+		name = self.client.service_path(self.project_id, service_name)
+		return self.client.get_service(name)
+
+	def testFrontendServiceExists(self):
+		""" Test that the Frontend Custom Service gets created. """
+		response = self.checkForService('frontend-srv')
+		# check that we found an object
+		self.assertTrue(response)
+
+class TestServiceSLO(unittest.TestCase):
+	def setUp(self):
+		self.client = monitoring_v3.ServiceMonitoringServiceClient()
+		self.project_id = getProjectId()
+
+	def checkForSLO(self, service, slo):
+		name = self.client.service_level_objective_path(self.project_id, service, slo)
+		return self.client.get_service_level_objective(name)
+
+	def testFrontendServiceSLOExists(self):
+		""" Test that for each service that two SLOs (availability, latency) get created. """
+		slos = ['availability-slo', 'latency-slo']
+		for slo in slos:
+			response = self.checkForSLO('frontend-srv', slo)
+			# check that we found an object
+			self.assertTrue(response)
+
+class TestSLOAlertPolicy(unittest.TestCase):
+	def setUp(self):
+		self.client = monitoring_v3.AlertPolicyServiceClient()
+
+	def checkForAlertingPolicy(self, policy_display_name):
+		policies = self.client.list_alert_policies(project_name)
+		for policy in policies:
+			if policy.display_name == policy_display_name:
+				return True
+		return False
+
+	def testFrontendServiceSLOAlertExists(self):
+		""" Test that the Alerting Policies for the Frontend Service SLO get created. """
+		found_availability_alert = self.checkForAlertingPolicy('Frontend Service Availability Alert Policy')
+		self.assertTrue(found_availability_alert)
```

Comment:
Why loop over the slos in the testFrontendServiceSLOExists and here just do them one at a time explicitly?

Reply:
Good point - there isn't a particular reason. I moved the earlier example to go over them one at a time explicitly since it will make it more clear as to where the point of failure is (if there is one).